sdk/communication/communication-identity/swagger/README.md

```diff
@@ -11,8 +11,8 @@ description: Communication identity client
 package-version: 1.3.2
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
-tag: package-2023-10
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/5797d78f04cd8ca773be82d2c99a3294009b3f0a/specification/communication/data-plane/Identity/readme.md
+tag: package-2025-04-01-preview
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/c7c2694ee53f22dfe56ee0d0fb9875ae5778d0f9/specification/communication/data-plane/Identity/readme.md
```

Comment:
we have to update the commit after the rest api PR merge

Reply:
Done